snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: lazygit
+title: Lazygit
+base: core24
+adopt-info: lazygit
+summary: simple terminal UI for git commands
+description: A simple terminal UI for git commands, written in Go with the gocui library.
+license: MIT
+grade: stable
+confinement: classic
+compression: lzo
+contact: 
+  - https://discord.gg/ehwFt2t4wt
+  - https://twitter.com/DuffieldJesse
+donation: https://github.com/sponsors/jesseduffield
+issues: https://github.com/jesseduffield/lazygit/issues
+source-code: https://github.com/jesseduffield/lazygit
+website: https://jesseduffield.com
```

Comment:
```suggestion
website: https://github.com/jesseduffield/lazygit
```

Reply:
That's the source-code. You want the website to the github repo too? You can verify yourself with the website.

Reply:
Thanks! Done!